src/Core/ELILO.pm

```diff
@@ -172,6 +172,7 @@ sub GetMetaData() {
 	noedd30		=> "bool:Don't force EDD30 Mode if not set:",
 	fpswa		=> "path:Specify the Filename for a specific FPSWA to load:",
 	"secure-boot"	=> "string:Support Secure Boot:auto",
+	multipath	=> "int:Number of additional multipath entries:0",
```

Comment:
Well, maybe this part is wrong https://github.com/openSUSE/perl-bootloader/blob/master/src/Core/ELILO.pm#L371
I think we should not throw away something we do not know. If we simple pass it thru it should not break anything and also do not remove customer own stuff there.